neurom/core/types.py

```diff
@@ -110,43 +118,38 @@ def __ne__(self, other):
         """Not equal operator."""
         return not self == other
 
-    @classmethod
-    def from_list(cls, values):
-        """Create a NeuriteType from a list of sub types."""
-        value = 0
-        for num, i in enumerate(values[::-1]):
-            new_value = cls._BASE**num * int(i)
-            value += new_value
-        obj = cls(value)
-        return obj
-
-    def to_list(self):
-        """Transform a NeuriteType or a positive integer in a list of sub types."""
-        digits = []
-        tmp_value = int(self)
-        while tmp_value:
-            digits.append(int(tmp_value % self._BASE))
-            tmp_value //= self._BASE
-        if not digits:
-            digits = [0]
-        res = digits[::-1]
-        if _ALL_SUBTYPE in res and len(res) > 1:
-            raise NeuroMError(
-                f"A subtype containing the value {_ALL_SUBTYPE} must contain only one element "
-                f"(current elements: {res})."
-            )
-        return res
-
```

Comment:
I think it would be interesting to have a public API to retrieve the subtypes. Since the `_subtypes` attribute is now a Tuple (so immutable), maybe we could just rename it into `subtypes` to make it public?

Reply:
Sure